Cabal/src/Distribution/PackageDescription/Check/Common.hs

```diff
@@ -1,3 +1,6 @@
+{-# LANGUAGE PatternSynonyms #-}
+{-# LANGUAGE ViewPatterns #-}
```

Comment:
Phil those are literally **not** needed here!

Are you sponsored by big-`ViewPatterns`? (smile)

Reply:
Yes, I missed removing those after I moved some code between modules. Thanks for noticing.

``` diff
$ git diff
diff --git a/.hlint.yaml b/.hlint.yaml
index 36c443fc7..7e3d69036 100644
--- a/.hlint.yaml
+++ b/.hlint.yaml
@@ -29,7 +29,6 @@
 - ignore: {name: "Redundant return"} # 9 hints
 - ignore: {name: "Replace case with fromMaybe"} # 6 hints
 - ignore: {name: "Replace case with maybe"} # 10 hints
-- ignore: {name: "Unused LANGUAGE pragma"} # 168 hints
 - ignore: {name: "Use $>"} # 5 hints
 - ignore: {name: "Use ++"} # 4 hints
 - ignore: {name: "Use :"} # 30 hints
```

```
$ hlint Cabal/src/Distribution/PackageDescription/Check/Common.hs
Cabal/src/Distribution/PackageDescription/Check/Common.hs:1:1-32: Warning: Unused LANGUAGE pragma
Found:
  {-# LANGUAGE PatternSynonyms #-}
Perhaps you should remove it.

Cabal/src/Distribution/PackageDescription/Check/Common.hs:2:1-29: Warning: Unused LANGUAGE pragma
Found:
  {-# LANGUAGE ViewPatterns #-}
Perhaps you should remove it.

2 hints
```

Reply:
Gone now.